src/components/CodeViewGridItem.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Card, GridItem, SkeletonText } from "@chakra-ui/react";
+import {
+  BlockLayout,
+  MakeCodeBlocksRendering,
+  MakeCodeProject,
+} from "@microbit-foundation/react-code-view";
+
+interface CodeViewGridItemProps {
+  project: MakeCodeProject;
+  gestureName: string;
+}
+
+const CodeViewGridItem = ({ project, gestureName }: CodeViewGridItemProps) => {
+  return (
+    <GridItem>
+      <Card
+        px={5}
+        h="120px"
+        display="flex"
+        borderColor="brand.500"
+        minW="400px"
+        width="fit-content"
+        justifyContent="center"
+      >
+        <Box
+          width={`${120 + gestureName.length * 5}px`}
+          py={2}
+          px={2}
+          overflow="hidden"
+        >
+          <MakeCodeBlocksRendering
+            code={project}
+            layout={BlockLayout.Clean}
+            loaderCmp={
+              <SkeletonText
+                w="full"
+                noOfLines={3}
+                spacing="5"
+                skeletonHeight="2"
+              />
+            }
+          />
+        </Box>{" "}
```

Comment:
I don't think the space is needed/wanted?

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/290/commits/4146305b8e9cf4ad1c27cca0e7622c9c1bef7e20